framework/src/modules/auth/schemas.ts

```diff
@@ -12,7 +12,15 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
+export interface AuthData {
+	nonce: number;
```

Comment:
`nonce`'s type is `uint64` in the schema, shouldn't it be a `bigint`? It's also `bigint` in the old `sequence` module.